src/pybind/mgr/dashboard/frontend/src/app/shared/directives/copy2clipboard-button.directive.ts

```diff
@@ -8,6 +8,8 @@ import { ToastrService } from 'ngx-toastr';
 export class Copy2ClipboardButtonDirective implements OnInit {
   @Input()
   private cdCopy2ClipboardButton: string;
+  @Input()
+  private formatted = 'no';
```

Comment:
A boolean would be better suited here. For example `isTextArea` or `isFormatted`.

Reply:
I wasn't able to get it working on the HTML file so that I can just add an attribute similar to `checked` or `readonly`.